tools/ack-discover/maintenance_phases.py

```diff
@@ -0,0 +1,23 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	 http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+NONE = "NONE"
+PREVIEW = "PREVIEW"
+GENERAL_AVAILABILITY = "GENERAL AVAILABILITY"
+DEPRECATED = "DEPRECATED"
```

Comment:
Enum? https://docs.python.org/3/library/enum.html :)

Reply:
ya ya... old Python fart here... :)

![bitmoji](https://sdk.bitmoji.com/render/panel/bf1be74f-fda9-40e3-aec1-4f79c8843eca-95be6062-5b42-43dc-bfe1-8b94033eaa9e-v1.png?transparent=1&palette=1&width=246)

Reply:
Yeah there are enums in Python whose values are strings - https://docs.python.org/3/library/enum.html#using-a-descriptive-string

But in general I agree that the enum implementation in general is not a great experience. In Go at least they're treated like strings, just with a more specific type.